addons/point_of_sale/static/src/xml/Popups/ClosePosPopup.xml

```diff
@@ -85,6 +93,7 @@
                 <div class="small button icon"
                     t-on-click="downloadSalesReport"
                     title="Download a report with all the sales of the current PoS Session">
+                        <div class="daily sale">Daily Sale</div>
```

Comment:
Where is the "printer" label/icon? The specs asks for it so that we can differentiate whether it will be printed or downloaded.

Reply:
The  "printer" label/icon is just below in the code, and it is only displayed if there is a printer connected.

Reply:
Oh I see. The specs is not asking for additional button. It's just asking for label.